CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
 
+## [Unreleased - header component PR]
```

Comment:
Why 2 `Unreleased` sections? Also, we should use things like `### Added`, `### Fixed` etc. like in https://keepachangelog.com/ 🙂

Reply:
Please full all changes from develop in your branch